patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+index 7c9bb358c9967e6c7c02b96e17764f2418df7979..f4a6b6ebf09bcb5e5209f39113dbb99937a0b8cb 100644
+--- a/src/main/java/net/minecraft/server/EntityInsentient.java
++++ b/src/main/java/net/minecraft/server/EntityInsentient.java
+@@ -675,11 +675,23 @@ public abstract class EntityInsentient extends EntityLiving {
+     }
+ 
+     public boolean b(ItemStack itemstack, ItemStack itemstack1) {
+-        return itemstack.getDamage() >= itemstack1.getDamage() && (!itemstack.hasTag() || itemstack1.hasTag()) ? (itemstack.hasTag() && itemstack1.hasTag() ? itemstack.getTag().getKeys().stream().anyMatch((s) -> {
+-            return !s.equals("Damage");
+-        }) && !itemstack1.getTag().getKeys().stream().anyMatch((s) -> {
+-            return !s.equals("Damage");
+-        }) : false) : true;
++        if (itemstack.getDamage() >= itemstack1.getDamage() && (!itemstack.hasTag() || itemstack1.hasTag())) {
++            if (itemstack.hasTag() && itemstack1.hasTag()) {
++                if (itemstack.getTag().getKeys().stream().noneMatch((s) -> {
++                    return !s.equals("Damage");
++                })) return false;
++                for (String s : itemstack1.getTag().getKeys()) {
++                    if (!s.equals("Damage")) {
++                        return (false);
++                    }
++                }
++                return (true);
++            } else {
++                return (false);
++            }
++        } else {
++            return true;
++        }
+     }
+ 
+     public boolean canPickup(ItemStack itemstack) {
+diff --git a/src/main/java/net/minecraft/server/EnumDirection.java b/src/main/java/net/minecraft/server/EnumDirection.java
+index 1aa9d9fd074c186b209696ee46e0dda181372f4b..315b4b6afa1870490200c602dc6f30ccbcf16d94 100644
+--- a/src/main/java/net/minecraft/server/EnumDirection.java
++++ b/src/main/java/net/minecraft/server/EnumDirection.java
+@@ -4,12 +4,8 @@ import com.google.common.collect.Iterators;
+ import com.mojang.serialization.Codec;
+ import it.unimi.dsi.fastutil.longs.Long2ObjectMap;
+ import it.unimi.dsi.fastutil.longs.Long2ObjectOpenHashMap;
+-import java.util.Arrays;
+-import java.util.Comparator;
+-import java.util.Iterator;
+-import java.util.Locale;
+-import java.util.Map;
+-import java.util.Random;
++
++import java.util.*;
+ import java.util.function.Predicate;
+ import java.util.stream.Collectors;
+ import java.util.stream.Stream;
+@@ -27,28 +23,57 @@ public enum EnumDirection implements INamable {
+     private final EnumDirection.EnumAxisDirection l;
+     private final BaseBlockPosition m;
+     private static final EnumDirection[] n = values(); private static final EnumDirection[] ALL = n;// Yatopia OBF HELPER
+-    private static final Map<String, EnumDirection> o = (Map) Arrays.stream(EnumDirection.n).collect(Collectors.toMap(EnumDirection::m, (enumdirection) -> {
```

Comment:
Streams for what is ran once for rather small sets of data is fine. This isn't a performance issue *at all*.